### ISSUE TITLE
<section> without labels or heading is not reported as an issue

### ISSUE THREAD

**Issue by Yaffle** [dequelabs/axe-core#1830]
A demo page:
```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>My test page</title>
  </head>
  <body>
    <main>
      <h1>Test page</h1>
      <section>
        <div>Section without labels</div>
      </section>
      <section>
        <div>Another section without labels</div>
      </section>
    </main>
  </body>
</html>
```
The section element without heading/labels makes no sense, seems, as it will not be used as a landmark.
It would be nice if "axe" could detect such issues.

**Comment by straker** [comment]
Improper use of the `<section>` element isn't really an accessibility problem and a rule checking for them would be prone to false positives. As such, we probably won't create a rule for it. But if you're interested in having a rule do so you can use [`axe.configure` to set one up](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#api-name-axeconfigure).

**Comment by Yaffle** [comment]
>>Improper use of the <section> element isn't really an accessibility problem
It does not make sense it without heading.

I am a little surprised, that it is so easy to make a mistake when using <section> and you have no rule to check it and that you are closing the issue.

Why is it a problem to calculate the accessible name for <section>?

**Comment by straker** [comment]
The accessible name of a `<section>` element is based on its `aria-label` or `aria-labelledby`, and not on a heading element. A `<section>` without a one of these is essentially a `<div>` element as it presents no semantic information to a screen reader (it has no [`role`](https://www.w3.org/TR/html-aria/#section) and since there [isn't a thing as the document outline](http://adrianroselli.com/2016/08/there-is-no-document-outline-algorithm.html), doesn't contribute to anything semantic).

Lastly, section elements do [not necessarily have to have a heading](https://html.spec.whatwg.org/multipage/sections.html#the-section-element) and that's why we are concerned about false positives.
